deploy/auto-install.sh

```
@@ -120,15 +117,15 @@ setup_docker_openwisp() {
 		echo -ne ${GRN}"(5/5) Use Let's Encrypt SSL? (y/N, blank for no): "${NON}
 		read use_letsencrypt
 	else
-		cp $env_path $ENV_USER &>>$LOG_FILE
+		cp "$env_path" "$ENV_USER" &>>"$LOG_FILE"
 	fi
 	echo ""
 
 	download_docker_openwisp "$openwisp_version"
 
-	cd $INSTALL_PATH &>>$LOG_FILE
+	cd "$INSTALL_PATH" &>>"$LOG_FILE"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)